[Bug]: Cannot upgrade due to jest-environment-jsdom requirement

### Version

15.0.0

### Steps to reproduce

Install version 15.0.0 on an angular 20.3.1 project

### Expected behavior

Installation is successful 

### Actual behavior

I have an error: 
```

  npm error code ERESOLVE
  npm error ERESOLVE could not resolve
  npm error
  npm error While resolving: urbantz@1.0.0
  npm error Found: jest-environment-jsdom@29.7.0
  npm error node_modules/jest-environment-jsdom
  npm error   peerOptional jest-environment-jsdom@"^29.5.0" from @angular-devkit/build-angular@20.3.0
  npm error   node_modules/@angular-devkit/build-angular
  npm error     dev @angular-devkit/build-angular@"20.3.0" from the root project
  npm error     peer @angular-devkit/build-angular@">=18.0.0 < 21.0.0" from @storybook/angular@9.1.4
  npm error     node_modules/@storybook/angular
  npm error       dev @storybook/angular@"9.1.4" from the root project
  npm error
  npm error Could not resolve dependency:
  npm error dev jest-preset-angular@"15.0.0" from the root project
  npm error
  npm error Conflicting peer dependency: jest-environment-jsdom@30.1.2
  npm error node_modules/jest-environment-jsdom
  npm error   peer jest-environment-jsdom@"^30.0.0" from jest-preset-angular@15.0.0
  npm error   node_modules/jest-preset-angular
  npm error     dev jest-preset-angular@"15.0.0" from the root project
  npm error
  npm error Fix the upstream dependency conflict, or retry
  npm error this command with --no-strict-peer-deps, --force, or --legacy-peer-deps
  npm error to accept an incorrect (and potentially broken) dependency resolution.
  npm error

```

### Additional context

See reference: https://github.com/angular/angular-cli/issues/30760

We cannot upgrade this lib to version 15 as long as angular requires  jest-environment-jsdom@"^29.5.0" 
Is there a possibility to be more flexible on the requirements of this lib on jest-environment-jsdom
? 

Thank you 



### Environment

```shell
Linux ubuntu
```

--- a/package.json
+++ b/package.json
@@ -64,7 +64,6 @@
     "@angular/platform-browser": ">=18.0.0 <21.0.0",
     "@angular/platform-browser-dynamic": ">=18.0.0 <21.0.0",
     "jest": "^30.0.0",
-    "jest-environment-jsdom": "^30.0.0",
     "jsdom": ">=26.0.0",
     "typescript": ">=5.5"
   },

--- a/website/docs/getting-started/installation.md
+++ b/website/docs/getting-started/installation.md
@@ -11,7 +11,7 @@ import TabItem from '@theme/TabItem';
 You can install `jest-preset-angular` and dependencies all at once with one of the following commands.
 
 ```bash npm2yarn
-npm install -D jest jest-preset-angular @types/jest
+npm install -D jest jest-preset-angular @types/jest jest-environment-jsdom jsdom
 ```
 
 ### Configuration

--- a/website/versioned_docs/version-14.x/getting-started/installation.md
+++ b/website/versioned_docs/version-14.x/getting-started/installation.md
@@ -11,7 +11,7 @@ import TabItem from '@theme/TabItem';
 You can install `jest-preset-angular` and dependencies all at once with one of the following commands.
 
 ```bash npm2yarn
-npm install -D jest jest-preset-angular @types/jest
+npm install -D jest jest-preset-angular @types/jest jest-environment-jsdom jsdom
 ```
 
 ### Configuration

--- a/website/versioned_docs/version-15.0/getting-started/installation.md
+++ b/website/versioned_docs/version-15.0/getting-started/installation.md
@@ -11,7 +11,7 @@ import TabItem from '@theme/TabItem';
 You can install `jest-preset-angular` and dependencies all at once with one of the following commands.
 
 ```bash npm2yarn
-npm install -D jest jest-preset-angular @types/jest
+npm install -D jest jest-preset-angular @types/jest jest-environment-jsdom jsdom
 ```
 
 ### Configuration

